src/arrays/referencing_array_and_its_elements/hard/q3/EvenorOdd.java

```diff
@@ -0,0 +1,59 @@
+package arrays.referencing_array_and_its_elements.hard.q3;
+/*
+Write a Java program that input an array of 10 integers with positive numbers.
```

Comment:
For this, I think we should change the question text to be this:
`Write a java program that accepts an array of 10 numbers from the user and then creates a new array of only even numbers from the original array. To do this, create a method named even that accepts an integer array, counts the number of even integers in the array, and then transfers them to a new array of that length. Then, return the new array to the main method, and print out its elements. `

Reply:
As such, you will have to update the test